### PR TITLE
MINOR: Remove version check from `test_command_help`

### PR DESCRIPTION
**Pull request by viirya**
# Which issue does this PR close?

Closes #.

# Rationale for this change
 
 

It causes test failure in CI after version upgrading:

```
---- tests::test_command_help stdout ----
thread 'tests::test_command_help' panicked at 'assertion failed: `(left == right)`
  left: `"parquet 15.0.0
  ...
   right: `"parquet 16.0.0
```

Seems not necessary to have a version there and ask for updating the file each time.

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

**Comment by viirya** [comment]
cc @tustvold

File: parquet/src/bin/parquet-fromcsv.rs [hunk 1]
```diff
@@ -403,7 +403,9 @@ mod tests {
         let mut buffer_vec = Vec::new();
         let mut buffer = std::io::Cursor::new(&mut buffer_vec);
         cmd.write_long_help(&mut buffer).unwrap();
-        let actual = String::from_utf8(buffer_vec).unwrap();
+        let mut actual = String::from_utf8(buffer_vec).unwrap();
```
Comment (tustvold) [inline parquet/src/bin/parquet-fromcsv.rs:406]: Maybe a comment explaining what this is doing?

**Review by tustvold — APPROVED** [review]
Thank you

File: parquet/src/bin/parquet-fromcsv.rs (hunk 1 above)
Comment (viirya) [inline parquet/src/bin/parquet-fromcsv.rs:406]:
```suggestion
        // Remove Parquet version string from the help text
        let mut actual = String::from_utf8(buffer_vec).unwrap();
```

**Comment by viirya** [comment]
Thanks @tustvold

**Pull request merged**